pkg/cmd/cli/cmd/newapp.go

```diff
@@ -139,7 +140,7 @@ func RunNewApplication(f *clientcmd.Factory, out io.Writer, c *cobra.Command, ar
 		case *kapi.Service:
 			fmt.Fprintf(c.Out(), "Service %q created at %s:%d to talk to pods over port %d.\n", t.Name, t.Spec.PortalIP, t.Spec.Port, t.Spec.TargetPort.IntVal)
 		case *buildapi.BuildConfig:
-			fmt.Fprintf(c.Out(), "A build was created - you can run `osc start-build %s` to start it.\n", t.Name)
+			fmt.Fprintf(c.Out(), "A build was created - you can run `osc start-build %s -n %s` to start it.\n", t.Name, namespace)
```

Comment:
Most users will be in their default namespace, so this is too aggressive.  Make it an if == defaultNamespace